pkgs/development/libraries/pango/default.nix

```
@@ -39,6 +39,7 @@ in stdenv.mkDerivation rec {
 
   mesonFlags = [
     "-Dgtk_doc=${if stdenv.isDarwin then "false" else "true"}"
+    "-Dxft=${if stdenv.isDarwin then "disabled" else "enabled"}"
     "-Dlibthai=disabled" # Not packaged in Nixpkgs
   ];
```

to avoid mass linux rebuilds
also maybe a comment as to why this is needed
```suggestion
    "-Dgtk_doc=${if stdenv.isDarwin then "false" else "true"}"
    "-Dxft=${if stdenv.isDarwin then "disabled" else "enabled"}"
    "-Dlibthai=disabled" # Not packaged in Nixpkgs
  ] ++ lib.optionals stdenv.isDarwin [
    "-Dxft=disabled"  # only works with x11
  ];
```